static/js/containers/LearnerPage.js

```diff
@@ -39,6 +39,7 @@ class LearnerPage extends React.Component<*, LearnerPageProps, *> {
     const { params: { username }, fetchProfile } = this.props;
     fetchProfile(username);
     this.fetchDashboard();
+    document.title = "Learner's Profile";
```

Comment:
Is it possible to include the Learner's Full name here? i.e. `[Preferred Name] [Last Name]'s MicroMasters Profile`